<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomDialog</class>
 <widget class="QDialog" name="RandomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="attackLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="attackDistinctCheckBox">
         <property name="text">
          <string>Distinct</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="defenceLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="defenceDistinctCheckBox">
         <property name="text">
          <string>Distinct</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>175</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="generatePushButton">
     <property name="text">
      <string>&amp;Generate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>generatePushButton</sender>
   <signal>clicked()</signal>
   <receiver>RandomDialog</receiver>
   <slot>generatePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generatePressed()</slot>
 </slots>
</ui>
